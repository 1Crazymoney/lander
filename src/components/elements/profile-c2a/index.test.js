import React from 'react';
import ProfileC2a from './index';
import renderer from 'react-test-renderer';


it('Render', () => {

  const props = {};

  const component = <ProfileC2a {...props} />;

  const tree = renderer
    .create(component)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
