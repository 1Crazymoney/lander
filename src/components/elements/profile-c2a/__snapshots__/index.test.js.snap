// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render 1`] = `
<a
  className="profile-c2a"
  href="/"
>
  <img
    alt="Logo"
    className="logo"
    src="lander-512.png"
  />
</a>
`;
